rero_ils/modules/notifications/api.py

```diff
@@ -67,33 +71,130 @@ class Notification(IlsRecord):
     provider = NotificationProvider
     model_cls = NotificationMetadata
 
-    @property
-    def organisation_pid(self):
-        """Get organisation pid for notification."""
-        location_pid = self.loan.get('transaction_location_pid')
-        location = Location.get_record_by_pid(location_pid)
-        return location.organisation_pid
+    def dispatch(self, delay=True):
+        """Dispatch notification."""
+        self = Dispatcher().send(notification=self)
+        return self
+
+    def update_process_date(self):
+        """Update process date."""
+        self['process_date'] = datetime.now().isoformat()
+        self.update(data=self.dumps(), dbcommit=True, reindex=True)
+        return self
+
+    def dumps_pids(self):
```

Comment:
Can you gives more informations in the documentations. And perhaps change the name: it suggest to dump a list of pids. Something like: resolve_refs or dumps_with_refs_resolution etc.

Reply:
Changed to `replace_pids_and_refs`